package.json

```diff
@@ -235,6 +235,6 @@
   },
   "license": "Artistic-2.0",
   "engines": {
-    "node": ">=10"
+    "node": "^12.13.0 || ^14.15.0 || >=16"
```

Comment:
Conditional exports weren't added until 12.17, and pattern trailers 12.20; `imports` was added in 12.19 - in case you want to raise the threshold farther.

Reply:
I think "when that version entered LTS" is the least distruptive and easiest to argue for.